lightkurve/seismology/core.py

```diff
@@ -578,9 +584,15 @@ def estimate_deltanu(self, method='acf2d', numax=None):
 
         if method == "acf2d":
             from .deltanu_estimators import estimate_deltanu_acf2d
+
+            fs = np.mean(np.diff(self.periodogram.frequency))
+            if not np.isclose(np.median(np.diff(self.periodogram.frequency.value)), fs.value):
+                raise ValueError("the ACF 2D method requires that the periodogram "
+                                 "has a grid of uniformly spaced frequencies.")
+
             result = estimate_deltanu_acf2d(self.periodogram, numax=numax)
 
-        self.deltanu = result
+        self.deltanu = resultq
```

Comment:
I guess this is a typo -- the unit tests fail here with `E       NameError: name 'resultq' is not defined`.

Reply:
Careless of me-- fixed!